modules/functions.js

```diff
@@ -108,30 +108,58 @@ module.exports = (client) => {
       command = client.commands.get(client.aliases.get(commandName));
     }
     if (!command) return `The command \`${commandName}\` doesn"t seem to exist, nor is it an alias. Try again!`;
-  
+
     if (command.shutdown) {
       await command.shutdown(client);
     }
     delete require.cache[require.resolve(`../commands/${commandName}.js`)];
     return false;
   };
 
+  client.loadBackgroundWorker = (workerName) => {
+    const worker = require(`../background/${workerName}`);
+    client.log("log", `Loading Background Worker: ${worker.conf.slug}. 👌`);
```

Comment:
Please change this to ``client.logger.log(`Loading Background Worker: ${worker.conf.slug}. 👌`);``